notebooks/_06-Architecture.md

```diff
@@ -21,33 +21,44 @@ jupyter:
+ 4. The cleaned dataset will be used in a function that estimated the distributions for variables of interest, this function will produce a parameter dictionary containing the type of variables, the best fitted distribution, and that distribution parameters.
+ 5. The resulting parameter dictionary from the previous step will be used to simulate needed data
+ 6. Based on observed availability of allternatives, a function will use the cleaned dataset to simulate an availability matrix for alternatives for the simulated dataset
+ 7. The specified model from step 3 will be used to simulate choices for the simulated dataset.
+ 8. The dataset will be used to estimate a choice model and recover the necessary parameters. These parameters can be stored in a dictionary.
+ 9. Steps 5-8 will be repeated for N times (say 100) and the parameters from each iteration will be stored in a dictionary entry.
+ 10. These paramaters will be stored in a dictionary that will be used in a function to plot the original parameters used to simulate the choices (from step 3) along with any recovered parameters from step 9.
+ 11. One variable will be perturbed to simulate a potential external intervention. In this scenario, no other variables will be changes since all covariates are independent.
+ 12. Causal effects will be estimated using the models estimated in step 3 and 9.
+ 13. Plots will be produced to show the distribution of "naive" and "true" causal effects
 
 
 For a realistic causal graph with confounding:
- - Ingest data
- - Manipulate data as needed, the output of this step will be a cleaned dataset
- - Specify a choice model based on the data at hand, the specification should follow required conventions by third party dependencies
- - Specify the realistic causal graph with desired confounding structures. The causal graph will be saved as a causalgraphicalmodels object and can be saved as .png file
- - The cleaned dataset will be used in a function that simulates choice data based on the specified model, resulting in a dataset with simulated choices
- - The dataset will be used to estimate a choice model and recover the necessary parameters. These parameters can be stored in a dictionary.
- - The previous two steps will be repeated for N times (say 100) and the parameters from each iteration will be stored in a dictionary entry.
- - A function that takes in the parameters of the estimated model and results in the percentage of time that the true parameter is recovered.
- - We will report parameter recovery percentages separately for each parameter, then build on that with aggregate percentaged for a subset of parameters 
-   then for all parameters.
+ 1. Ingest data
+ 2. Manipulate data as needed, the output of this step will be a cleaned dataset
+ 3. Specify a choice model based on the data at hand, the specification should follow required conventions by third party dependencies
```

Comment:
These steps are only performed once correct? I'm of the impression that you reuse the same cleaned dataset and choice model, and "true parameters" estimated from that specified model and the original data for both the naive and realistic causal graphs right? If so, I would begin with step 4 below and remove the first three steps (or mention that the results of steps 1 - 3 from the naive scenario are reused.